rust-actix/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Added by cargo
```

Comment:
Should we do a global .gitignore instead? I have that in my PR. 
Just so we don't end up with 7 gitignore (1 per project)

Reply:
We can have one gitignore in the root folder + one inside each language. I think it is more organized this way. If we disagree let me know